services/app/src/static/style/gameScore.css

```diff
@@ -0,0 +1,9 @@
+.id {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  margin-block: -450px;
+  margin-left: -700px;
```

Comment:
Why we need these negative margins? I would suggest to avoid those, as they might introduce problems when we will be placing other elements on the screen.